firmware/development/wave_analyzer.cpp

```diff
@@ -203,49 +178,45 @@ static float getSignalPeriodMs(int index) {
 //}
 
 static void reportWave(Logging *logging, int index) {
-	if (readers[index].hw == NULL) {
+	if (!readers[index].isStarted) {
 		return;
 	}
-	if (readers[index].hw->started) {
-//	int counter = getEventCounter(index);
-//	debugInt2(logging, "ev", index, counter);
 
-		float dwellMs = getSignalOnTime(index);
-		float periodMs = getSignalPeriodMs(index);
-
-		appendPrintf(logging, "duty%d%s", index, DELIMETER);
-		appendFloat(logging, 100.0f * dwellMs / periodMs, 2);
+	float dwellMs = getSignalOnTime(index);
```

Comment:
I'm not sure why it thought this was a diff, all I did was unwrap it from an `if (...)`